db/models/applications.js

```diff
@@ -131,6 +131,14 @@ class PendingApplication extends Model {
             { where: { id: { [Op.eq]: id } } }
         );
     }
+    static lastUpdatedTime(applicationId) {
```

Comment:
Is this necessary? Could we use the regular find by primary key method?

Reply:
I just wanted to fetch the field value we needed rather than the entire record from db, especially when it contains a fairly big json object which we don't need. Was just thinking of a minor performance win.